ale/drivers/isis_spice_driver.py

```diff
@@ -0,0 +1,588 @@
+from glob import glob
+import os
+import struct
+import re
+
+import pvl
+import numpy as np
+import quaternion
+
+from ale import config
+from ale.drivers.base import Driver, Isis3
+
+def read_table_data(table_label, file):
+    """
+    Helper function to read all of the binary table data
+
+    Parameters
+    ----------
+    table_label : PVLModule
+        The ISIS table label
+    file : file
+        The ISIS cube file
+
+    Returns
+    -------
+    bytes
+      The binary portion of the table data
+    """
+    file.seek(label['StartByte']-1) # This -1 is straight out of ISIS
```

Comment:
file is reserved, maybe input or someone other something.  I think that the -1 is likely required because the byte offsets at 1 off from the actual read start. I had to do an identical thing on another PDS style label.